### ISSUE TITLE
train accuracy decreasing with adding of data

### ISSUE THREAD

**Issue by kishore0905** [alfredfrancis/ai-chatbot-framework#105]
hello sir how can i remove overfitting by increasing the train accuracy some how i managed to decrease the loss in dataset but train accuracy is decreasing with increasing of training data.i am a beginner in A.I Can you explain me how to do that
